src/config.ts

```diff
@@ -4,38 +4,21 @@ import {homedir} from 'os'
 import * as path from 'path'
 
 let configDir
-if (process.env.HMD_CLI_CONFIG_DIR || process.env.CMD_CLI_CONFIG_DIR) {
-  configDir = process.env.HMD_CLI_CONFIG_DIR || process.env.CMD_CLI_CONFIG_DIR || ''
+if (process.env.HMD_CLI_CONFIG_DIR) {
+  configDir = process.env.HMD_CLI_CONFIG_DIR || ''
 } else {
   configDir = path.join(homedir(), '.hackmd')
 }
 
 const configFilePath = path.join(configDir, 'config.json')
-const defaultCookiePath = path.join(homedir(), '.hackmd', 'cookies.json')
 
 const defaultConfig = {
-  cookiePath: defaultCookiePath,
-  serverUrl: 'https://hackmd.io',
-  enterprise: true
-}
-
-function toBooleanConfig(configValue?: string | boolean) {
-  if (configValue && typeof configValue === 'string') {
-    return (configValue === 'true')
-  }
-  return configValue
+  hackmdAPIEndpointURL: 'https://api.hackmd.io/v1'
 }
 
 const envConfig = {
-  cookiePath: process.env.HMD_CLI_COOKIE_PATH || process.env.CMD_CLI_COOKIE_PATH,
-  serverUrl: process.env.HMD_CLI_SERVER_URL || process.env.CMD_CLI_SERVER_URL,
-  enterprise: (process.env.HMD_CLI_COOKIE_PATH || process.env.HMD_CLI_SERVER_URL)
-    ? true
-    : (process.env.CMD_CLI_COOKIE_PATH || process.env.CMD_CLI_SERVER_URL)
-      ? false
-      : toBooleanConfig(process.env.HMD_CLI_ENTERPRISE),
-  loginID: process.env.HMD_CLI_ID || process.env.CMD_CLI_ID,
-  loginPassword: process.env.HMD_CLI_PASSWORD || process.env.CMD_CLI_PASSWORD
+  hackmdAPIEndpointURL: process.env.HMD_API_ENDPOINT_URL,
+  accesstoken: process.env.HMD_API_ACCESS_TOKEN
```

Comment:
`accesstoken` should be written in camelCase:

```
accessToken
```

Reply:
Fixed.